templates/include/telemetry/environment.1.parquetmr.txt

```diff
@@ -73,6 +73,7 @@ optional group environment {
       }
       optional boolean DWriteEnabled;
       optional boolean D2DEnabled;
+      optional boolean Headless;
```

Comment:
Drive-by: I think this change is not compatible with schema evolution in parquet (change to the fields in a nested struct).

Reply:
Is there a way I can test that assumption?

(( or I could just drop it. It's only a nice-to-have ))

Reply:
I'm not sure if there's an easy way to test it, so +1 for dropping :-/